tensorflow_lite_support/ios/task/processor/utils/sources/TFLDetectionUtils.m

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#import "tensorflow_lite_support/ios/task/processor/utils/sources/TFLDetectionUtils.h"
+
+@implementation TFLDetectionUtils
+
++ (TFLDetectionResult *)detectionResultFromCDetectionResults:
```

Comment:
Both `TFLDetectionUtils` and `TFLClassificationUtils` only contains code to convert from C object to ObjC object. Therefore I think you shouldn't create these util classes but instead using category to add the conversion code similarly to `TFLBaseOptions (Helpers)`